test/e2e/operator/nmstate_install_test.go

```diff
@@ -37,14 +38,43 @@ import (
 )
 
 var _ = Describe("NMState operator", func() {
+	DescribeTable("for control-plane size",
+		func(controlPlaneSize int) {
+			if isKubevirtciCluster() {
+				for i := 2; i <= controlPlaneSize; i++ {
```

Comment:
Do you think It would be clearer for a reader if the indexing was from 0 with a local func `nodeNameWIthOffset(num int) string`, that would add `2` ?

I think it would be easier to realize how many times this code loops.
Another option, since we only currently loop once when the PlaneSize is 2, a simple `if` may also be enough, but if you want to keep it general, it's fine

Reply:
I will refactor it a little, it's true that it's not very nice for readers

Reply:
It cannot start from 0 since node01 is already part of the control-plane and we don't want to re-label it.

Reply:
I can generalize this completly looking for not control plane and marking them as such